internal/pkg/metrics/metrics.go

```diff
@@ -0,0 +1,70 @@
+package metrics
+
+import (
+	"context"
+	"time"
+
+	policiesv1alpha2 "github.com/kubewarden/kubewarden-controller/apis/policies/v1alpha2"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/exporters/otlp"
+	"go.opentelemetry.io/otel/exporters/otlp/otlpgrpc"
+	"go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/metric/global"
+	controller "go.opentelemetry.io/otel/sdk/metric/controller/basic"
+	processor "go.opentelemetry.io/otel/sdk/metric/processor/basic"
+	"go.opentelemetry.io/otel/sdk/metric/selector/simple"
+)
+
+const (
+	METER_NAME                        = "kubewarden"
+	POLICY_COUNTER_METRIC_NAME        = "kubewarden_policy_total"
+	POLICY_COUNTER_METRIC_DESCRIPTION = "How many policies are installed in the cluster"
+)
+
+func New(openTelemetryEndpoint string) error {
+	ctx := context.Background()
+
+	driver := otlpgrpc.NewDriver(
+		otlpgrpc.WithInsecure(),
+		otlpgrpc.WithEndpoint(openTelemetryEndpoint),
+	)
```

Comment:
I got this default configuration ( `otlpgrpc.WithInsecure()` )from the OpenTelemetry examples. Which means it does not use secure connections. I was thinking to add more configuration for this. What do you think? Does it make sense add a secure connection to a sidecar collector? (considering that the user is using a sidecar)

Even if we decided to use sidecar as recommended way, the user can change how to deploy the OpenTelemetry. Which can cause issues... So, I believe it is still valuable add more confiuration here.